funnel/cli/periodic/stats.py

```diff
@@ -149,6 +149,15 @@ async def matomo_response_json(
 
 
 async def matomo_stats(date: str = 'yesterday') -> MatomoData:
```

Comment:
`date` as a parameter is useless now given everything else is hardcoded in the function. Might as well remove from here and put it in the body as `date = 'yesterday'`